.github/workflows/test.yml

```diff
@@ -30,11 +30,18 @@ jobs:
     steps:
     - uses: actions/checkout@v2
 
+    - name: Install GCC 10.2 Linux
+      if: ${{ runner.os == 'Linux' }}
+      uses: egor-tensin/setup-gcc@v1
+      with:
+          version: 10
```

Comment:
Name says its 10.2, but specification if more general 10.x

Reply:
True, will update the name

Reply:
Add a comment, "some C++20 features are not wortking on current gcc (ver) on gha, therefore we need 10.2)